website/app/controllers/show.js

```diff
@@ -9,12 +9,32 @@ import { inject as service } from '@ember/service';
 import { showdownConfig } from '../shared/showdown-config';
 
 export default class ShowController extends Controller {
+  queryParams = [
+    {
+      selectedTab: 'tab',
+    },
+    'searchQuery',
+    'selectedIconSize',
+  ];
+
   @service fastboot;
 
   @tracked sections = A([]);
   @tracked tabs = A([]);
   @tracked tocs = A([]);
 
+  get selectedTabIndex() {
+    if (!this.selectedTab) {
+      return 0;
+    }
+
+    let tab = this.tabs.find((el) => {
+      // trim trailing slashes from query param
```

Comment:
I see in the icons controller you're using this
```
this.router.currentRoute.queryParams['searchQuery']
```
which returns directly the query param without the trailing slash (I imagine), could something similar also be used in this context?

Reply:
it turns out the trailing slash was from previous work I had done and it is no longer present, so I just removed that part